src/__tests__/App.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { MemoryRouter as Router } from "react-router-dom";
+import { render, cleanup, fireEvent } from "@testing-library/react";
+import App from "../App";
+
+beforeEach(() => cleanup);
+
+describe("<App/>", () => {
+    it("should render App", () => {
+        render(
+          <Router>
+            <App/>
+          </Router>
+        );
+       
```

Comment:
I dont see anything tested here. You need to test against the expected behaviour